src/ds-style-fix.scss

```diff
@@ -663,3 +663,16 @@ div.contact-button {
   margin-left: auto;
   margin-right: auto;
 }
+
+.private-lock {
+  color: #487980;
+}
+
+.accordion-hr {
```

Comment:
Rename to .my-1 or .my-2, etc for when we ever switch to https://v4-alpha.getbootstrap.com/utilities/spacing/